config.toml

```diff
@@ -3,6 +3,7 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 copyright = "Cloud Native Computing Foundation"
 
 enableGitInfo = true
+theme = "oci-tailwind"
```

Comment:
We *could* do this as a theme, but personally I find that creating a theme makes the directory structure trickier to navigate without providing much benefit.

If you'd like the theme to be reusable across projects, however, that's a different story!

Reply:
I'll have a think on it. I like the idea of doing overrides to a default in a seperate location but as I continue I'm finding weird idiosyncronisities with the approach – namely that configuring `tailwind.config.js` only seems to work in the theme dir and not the main hugo dir's.